fabric_plugin/tests/test_fabric_plugin.py

```diff
@@ -54,7 +54,8 @@ def _execute(self,
                  process=None,
                  ip=None,
                  custom_input='value',
-                 use_sudo=False):
+                 use_sudo=False,
+                 non_recoverable_errors=[]):
```

Comment:
Please do not use immutable types as default values. 
```suggestion
                 non_recoverable_errors=None):
                 non_recoverable_errors = non_recoverable_errors or []
```